### ISSUE TITLE
Error in pure contract example?

### ISSUE THREAD

**Issue by chipshort** [onrirr/c3-web#11]
This [line here](https://github.com/c3lang/c3-web/blob/e91e1e9ae67726643ae19116a23f3ca5f4deb8d9/src/content/docs/references/docs/contracts.md?plain=1#L118) looks like it should be a call to `foo()` instead. There is no`SecretFunc` defined anywhere.

**Comment by lerno** [comment]
Yes, thank you, I'll fix it.

**Comment by lerno** [comment]
Wow, it looks really broken.

**Comment by lerno** [comment]
Does it look better now?

**Comment by chipshort** [comment]
Now both examples under "Pure in detail" look the same except that one claims "The compiler cannot reason about this!" while the other claims it throws an error. Which one is it?

**Comment by corsinmusic** [comment]
Yeah, I also just noticed that both examples are the same except for the comment on line 15.

**Comment by lerno** [comment]
Updated again...

**Comment by ManuLinares** [comment]
both problems have already been fixed
